src/index.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {createStore} from 'redux';
-import {Provider} from 'react-redux';
 import App from './App';
-import appReducer from './reducers/appReducer';
-
-const store = createStore(appReducer); //optional parameter  window.STATE_FROM_SERVER
 
 const rootNode = document.getElementById("react-root");
-
+//render the whole app @root node
 ReactDOM.render(
-  <Provider store={store}>
```

Comment:
The Provider will be needed here to pass store?

Reply:
yes just refactored the code after adding material ui theme